### ISSUE TITLE
WCAG 2.2 typo and broken link on 2.5.5 Failure

### ISSUE THREAD

**Issue by jamieherrera** [w3c/wai-wcag-quickref#243]
Page: How to Meet, #Target size: https://www.w3.org/WAI/WCAG22/quickref/#target-size

Failure section says "2.5.3" but the issue for Target Size is 2.5.5.

Related (can be a separate ticket):
https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html
the link to 2.5.5 Target Size (enhanced) is broken and goes to a 404 page.

**Comment by iadawn** [comment]
Seems to have been fixed at some point
